Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -2140,6 +2140,9 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
 
     async _handleEventBadgeClicked(event)
     {
+        if (this._eventBadgePopover)
+            return; // A popover will auto-dismiss when clicking outside it. The trigger is outside the popover. The instance is cleared by `didDismissPopover()` called by the popover on its delegate.
```

Comment:
IMO this comment is unnecessary.  I already read this `if` as "don't attempt to show the popover if we're already showing one", which is enough.